pkg/apis/events/v1beta1/conversion.go

```diff
@@ -56,3 +64,31 @@ func Convert_core_Event_To_v1beta1_Event(in *k8s_api.Event, out *v1beta1.Event,
 	out.DeprecatedCount = in.Count
 	return nil
 }
+
+func AddFieldLabelConversionsForEvent(scheme *runtime.Scheme) error {
+	return scheme.AddFieldLabelConversionFunc(SchemeGroupVersion.WithKind("Event"),
+		func(label, value string) (string, string, error) {
+			switch label {
```

Comment:
@wojtek-t, PTAL to make sure these make sense

Reply:
regarding <-> involvedObject is fine: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/events/v1beta1/conversion.go#L30

other comments below